nixos/modules/services/networking/avahi-daemon.nix

```diff
@@ -275,6 +275,7 @@ in
         BusName = "org.freedesktop.Avahi";
         Type = "dbus";
         ExecStart = "${pkgs.avahi}/sbin/avahi-daemon --syslog -f ${avahiDaemonConf}";
+        ConfigurationDirectory = concatStringsSep " " [ "avahi" "avahi/services" ];
```

Comment:
What's the difference between this and
```
        ConfigurationDirectory = [ "avahi" "avahi/services" ];
```

Reply:
My version results in "ConfigurationDirectory=avahi avahi/services" which was the syntax I found in systemd-docs, your version evaluates and creates two entries:

```
ConfigurationDirectory=avahi 
ConfigurationDirectory=avahi/services
```

I was unable to find another package having multiple items at this option but as services is just a sub-directory I just it with just this one and seems to work as well.